projects/gnupg/fuzz_decrypt.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+max_len = 65536
```

Comment:
This won't work, please see https://github.com/google/oss-fuzz/blob/730451d8714ff9b5421e4c1a5ea57535dd62fcad/docs/new_project_guide.md#custom-libfuzzer-options-for-clusterfuzz

Reply:
This is not where the limit gets enforced.

The limit gets enforced in fuzz_import.c : https://github.com/google/oss-fuzz/pull/1871/files#diff-49878ea89d196f56b924c50f82358f15R135

Reply:
Cool! Could you please remove this line from `.options` files then?

Reply:
Isn't it somehow useful for libfuzzer ?

Reply:
It is, unless you enforce the limits in the target itself, which is more universal and preferred approach.

Reply:
That makes sense to remove this when this limit is in the source code:-)

What I meant by putting this options file is to tell the fuzzer : "I do not think that you will find more bugs with longer inputs, but try if you wish"

Reply:
Yes, you're right about this, but on our infrastructure it might be helpful during the very first runs only, and at the same point it still might be interesting to let the fuzzer at least try something long :)

Either way, we're trying not to land such `.options` since we do discourage using that approach.

Thanks for removing!